src/wasm-binary.h

```diff
@@ -1119,8 +1121,8 @@ class WasmBinaryBuilder {
         e->value = name;
```

Comment:
if name is not set, then we are setting that null string here, which is bad.

we would have to autogenerate a name earlier, in parallel to the `getString`.